winbuild/build_prepare.py

```diff
@@ -131,7 +131,7 @@ def cmd_msbuild(
 DEPS: dict[str, dict[str, Any]] = {
     "libjpeg": {
         "url": f"{SF_PROJECTS}/libjpeg-turbo/files/{V['JPEGTURBO']}/"
-        f"libjpeg-turbo-{V['JPEGTURBO']}.tar.gz/download",
+        "FILENAME/download",
```

Comment:
This string probably fits on a single line now.

Reply:
You're right. I've pushed a commit.